src/plugins/data/public/ui/query_string_input/query_string_input.tsx

```diff
@@ -60,6 +60,7 @@ interface Props {
   onChangeQueryInputFocus?: (isFocused: boolean) => void;
   onSubmit?: (query: Query) => void;
   dataTestSubj?: string;
+  dropdownHeight?: string;
```

Comment:
I think it would be better if we could override this setting via CSS in visualize editor, rather than pass it in as an argument, or at least somehow resolve this by passing in a custom class.
The `QueryStringInput` component is public API and I'd rather not add very specific properties to it.